src/dkg.ts

```diff
@@ -102,7 +102,7 @@ export class DkgClient {
     return {
       id: ritualId,
       dkgPublicKey: DkgPublicKey.fromBytes(dkgPkBytes),
-      threshold: ritualParams.threshold,
+      threshold: ritual.dkgSize, // TODO: dkgSize is not the threshold, but in this case shares == threshold == dkgSize
```

Comment:
I'm confused by this line...could you clarify what's going on here? Is this testnet specific code?

Reply:
Yes, it's a testnet-specific code. I added clarification.